### ISSUE TITLE
User Agent spoofing

### ISSUE THREAD

**Issue by raidancampbell** [cwruacm/lulu#18]
Lulu's HTML title response (when you post a link, this is Lulu's response), commonly comes across errors (406, 403, etc).  When I click the link, I am transferred to the expected site without error.

This is likely because the user agent that Lulu uses to visit these sites indicates it is a bot in some way.  A simple solution would be to change the user agent that Lulu sends in its HTTP header.

**Comment by raidancampbell** [comment]
Turns out @Aaronneyer already has done this, but [the user agent looks to be around 2008](http://en.wikipedia.org/wiki/History_of_Firefox#Release_history): it was version 2.0.0.13, and the gecko release looks like a datecode

I'd suggest using

```
Mozilla/5.0 (Windows NT 6.1; WOW64; rv:33.0) Gecko/20120101 Firefox/33.0
```

or one of the other user agent strings from [this site](http://www.useragentstring.com/pages/Firefox/).
The file in question is [lulu/scripts/web.coffee](https://github.com/cwruacm/lulu/blob/master/scripts/web.coffee)

**Comment by brenns10** [comment]
Closed by #25.
